cmd/podmanV2/images/save.go

```diff
@@ -0,0 +1,92 @@
+package images
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	"github.com/containers/libpod/cmd/podmanV2/parse"
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/containers/libpod/pkg/util"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+const (
+	ociManifestDir  = "oci-dir"
```

Comment:
We can't put these somewhere common? I'm seeing a lot of magic strings from here floating around in the code

Reply:
@baude `registry` would be the package, if this is only for the CLI.  Otherwise, `entities` if it is needed more further afield.